<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>50</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; 
</string>
   </property>
   <property name="text">
    <string>Welcome to HexaFitness</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>115</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeVerCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 7px;
    padding: 7px;
    color: black;  /* Ensures the entered text color is black */
    background-color: white;  /* Ensures the background is white */
}

QLineEdit:placeholder {
    color: gray;  /* Ensures the placeholder text color is gray */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>User name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid gray;
    border-radius: 7px;
    padding: 7px;
    color: black; 
    background-color: white; 
}

QLineEdit:placeholder {
    color: gray; 
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#loginButton{	
	background-color :rgb(15, 211, 84);
	color: rgb(22, 17, 58);
	border-radius:13px;

}
QPushButton#loginButton:hover{	
	
	background-color: rgb(13, 190, 75);
}
QPushButton#loginButton:pressed{	
	padding-left:5px;
	padding-top:5px;
	background-color: rgb(10, 147, 58);
}
</string>
   </property>
   <property name="text">
    <string>L O G  I N</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>349</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label_3 {
	background-image: url(:/rrr/Untitled design.png);
	border-radius: 10px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/images/Untitled design.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; 
</string>
   </property>
   <property name="text">
    <string>User name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>334</x>
     <y>154</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; 
</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>242</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; 
</string>
   </property>
   <property name="text">
    <string>Don't have an account?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signupDButton">
   <property name="geometry">
    <rect>
     <x>449</x>
     <y>236</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; 
</string>
   </property>
   <property name="text">
    <string>Sign up here</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>loginButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>signupDButton</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
